chitchat/src/failure_detector.rs

```diff
@@ -181,10 +194,14 @@ impl SamplingWindow {
 
     /// Computes the sampling window's phi value.
     pub fn phi(&self) -> f64 {
-        // Ensure we don't call before any sample arrival.
-        assert!(self.intervals.mean() > 0.0 && self.last_heartbeat.is_some());
-        let elapsed_time = self.last_heartbeat.unwrap().elapsed().as_secs_f64();
-        elapsed_time / self.intervals.mean()
+        if self.last_heartbeat.is_none() {
```

Comment:
nit: Can we use an `if let Some(...)` or `let Some(..)` here? That way, I don't have to think about whether the call to `unwrap` is safe.